src/vumi2/routers.py

```diff
@@ -49,6 +50,12 @@ async def setup(self):
                 connector_name=name, outbound_handler=self.handle_outbound_message
             )
 
+        if self.config.default_app:
+            self.default_connector = await self.setup_receive_outbound_connector(
+                connector_name=self.config.default_app,
+                outbound_handler=self.handle_outbound_message,
+            )
```

Comment:
Can `default_app` be one of the apps in the mapping dict? If so, we need to check for that so we don't try to add a duplicate connector. If not, we should probably raise a validation error of some kind.